docker/README.md

```
@@ -0,0 +1,226 @@
+- `/IDENTIFY <account> <password>` — Log in
+- `/LOGOUT` — Log out
+- `/LISTACC [account]` — List accounts (oper only)
+- SASL PLAIN authentication support
+
+Account data is stored in `/home/unrealircd/unrealircd/data/obsidian-account.db`.
+
+Configure server icon via `ICON_URL` environment variable.
+
+### o-filehost Module
+Enables file hosting integration. Set `FILEHOST_URL` to your file hosting backend URL. When configured:
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)